tests/utils/fakeredis_ext.py

```diff
@@ -0,0 +1,115 @@
+""" Extensions for fakeredis, namely adding connection interface that is used by kombu """
+
+from collections import deque
+from itertools import count
+from fakeredis import FakeStrictRedis, FakePipeline
+
+
+class FakeStrictRedisWithConnection(FakeStrictRedis):
```

Comment:
Looks good overall. Just add a comment here to the affect of saying why we need to extend fake redis (since we are using commands not supported by fake redis and using the redis library at a lower level).